provisioner/registryv1/controllers/bundle_controller.go

```diff
@@ -199,7 +199,7 @@ func (r *BundleReconciler) ensureUnpackPod(ctx context.Context, bundle *olmv1alp
 	pod.SetNamespace(r.PodNamespace)
 
 	return util.CreateOrRecreate(ctx, r.Client, pod, func() error {
-		pod.SetLabels(map[string]string{"kuberpak.io/owner-name": bundle.Name})
+		pod.SetLabels(map[string]string{"core.rukpak.io/owner-name": bundle.Name})
```

Comment:
This can be done in a follow-up, but I think we should make this `core.rukpak.io` domain a constant, or that entire label a constant so we reduce the number of places we'd need to manually update it if we decide to go in another direction. Thoughts?

Reply:
Yes, we definitely need a consistent labeling strategy and have the label live in one place in the codebase. Good follow-up change.